dashboard/db/migrate/20171013180932_backfill_regional_partner_program_managers_from_contacts.rb

```diff
@@ -0,0 +1,16 @@
+class BackfillRegionalPartnerProgramManagersFromContacts < ActiveRecord::Migration[5.0]
+  def up
+    RegionalPartner.joins(:contact).each do |regional_partner|
```

Comment:
Why join, when we have [contact_id](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/regional_partner.rb#L8) already? We can just use that directly.

Unrelated nit: Since this [program_manager=](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/regional_partner.rb#L49) method takes an id not a user / program manager, it should be named `program_manager_id=`

Reply:
I thought it would be more railsy to assign the associations instead of directly manipulating the id attributes :)

I'll open a separate PR to rename `program_manager=` to 'program_manager_id=` and to update the controller that uses that method.